src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-dashboard.ts

```diff
@@ -446,6 +459,66 @@ class ZWaveJSConfigDashboard extends SubscribeMixin(LitElement) {
                   </p>
                 </div>
               </ha-card>
+              <ha-card>
+                <div class="card-header">
+                  <h1>
+                    ${this.hass.localize(
+                      "ui.panel.config.zwave_js.dashboard.nvm_backup.title"
+                    )}
+                  </h1>
+                </div>
+                <div class="card-content">
+                  <p>
+                    ${this.hass.localize(
+                      "ui.panel.config.zwave_js.dashboard.nvm_backup.description"
+                    )}
+                  </p>
+                </div>
+                <div class="card-actions">
+                  ${this._backupProgress !== undefined
+                    ? html`<ha-circular-progress
+                          size="small"
+                          .value=${this._backupProgress}
+                        ></ha-circular-progress>
+                        ${this.hass.localize(
+                          "ui.panel.config.zwave_js.dashboard.nvm_backup.downloading"
+                        )}
+                        ${(this._backupProgress * 100).toFixed(0)}%`
```

Comment:
```suggestion
                        ${(this._backupProgress}%`
```

do this calc on set `this._backupProgress` because of `ha-progres-ring`